api/src/main/java/org/pmiops/workbench/genomics/WgsCohortExtractionService.java

```diff
@@ -145,6 +155,15 @@ public TerraJob submitGenomicsCohortExtractionJob(DbWorkspace workspace, Long co
                     .methodConfigurationName(methodConfig.getName())
                     .useCallCache(false));
 
+    // Note: if this save fails we may have an orphaned job. Will likely need a cleanup task to
+    // check for such jobs.
+    DbWgsExtractCromwellSubmission dbSubmission = new DbWgsExtractCromwellSubmission();
+    dbSubmission.setSubmissionId(submissionResponse.getSubmissionId());
+    dbSubmission.setWorkspace(workspace);
+    dbSubmission.setCreator(userProvider.get());
+    dbSubmission.setSampleCount(personIds.size());
```

Comment:
note that this count does not reflect the # of participants that will be fetched by the job since the Cromwell job will filter out IDs that it can't find a match for.

Reply:
I think null or leaving out the column would be better. No data > incorrect data IMO